public/locales/uk/translation.json

```diff
@@ -26,6 +26,10 @@
     "DESCRIPTION": "Ви можете знайти проект, який підходить вам ідеально, і дізнатися, як ваша компанія може підтримати його.",
     "HEADING": "Соціальні та благодійні ініціативи України"
   },
+  "about": {
+    "TEXT": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum eu fugiat nulla pariatur.",
+    "HEADING": "Mеценат - <1>це <1><0></0></1></1> допомога"
```

Comment:
HEADING should go before TEXT (alphabetical order)

Reply:
Done!